tests/test_misc.py

```diff
@@ -652,7 +652,7 @@ def fn(x, a, b):
             self.assertTrue(same(fn(*args), correct))
             self.assertTrue(same(fn(*args), correct))
         self.assertEqual(cnts.frame_count, 1)
-        self.assertEqual(cnts.op_count, 1)
+        self.assertEqual(cnts.op_count, 2)
```

Comment:
Is the extra op because now we have a % b as part of graph?

Reply:
Correct!